runtime/rastrace/method_trace.c

```diff
@@ -311,22 +311,29 @@ traceMethodExitX(J9VMThread *thr, J9Method *method, UDATA isCompiled, void* exce
 
 }
 
-static void
-hookRAMClassLoad(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
+BOOLEAN
+setRAMClassExtendedMethodFlagsHelper(J9VMThread *thr, J9Class *clazz)
 {
-	J9VMInternalClassLoadEvent* event = (J9VMInternalClassLoadEvent *)eventData;
-	J9VMThread* thr = event->currentThread;
 	J9JavaVM *vm = thr->javaVM;
-	J9Class* clazz = event->clazz;
-	J9ROMClass* romClass = clazz->romClass;
-	U_32 i;
+	J9Method *method = clazz->ramMethods;
+	U_32 i = 0;
 
-	J9Method * method = clazz->ramMethods;
-	for (i = 0; i < romClass->romMethodCount; i++) {
+	for (i = 0; i < clazz->romClass->romMethodCount; i++) {
```

Comment:
cache `clazz->romClass->romMethodCount` to a variable

Reply:
Cached this, and also a few others within `trcengine.c:J9VMDllMain` since it is part of this PR.